content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -4,115 +4,142 @@ reviewers:
 - davidopp
 - lavalamp
 - liggitt
-title: Managing Service Accounts
-content_template: templates/concept
+title: Managing ServiceAccounts
+content_template: templates/task
 weight: 50
 ---
 
 {{% capture overview %}}
-This is a Cluster Administrator guide to service accounts. It assumes knowledge of
-the [User Guide to Service Accounts](/docs/user-guide/service-accounts).
 
-Support for authorization and user accounts is planned but incomplete.  Sometimes
-incomplete features are referred to in order to better describe service accounts.
+A _ServiceAccount_ provides an identity for processes that run in a Pod.
+
+A process inside a Pod can use the identiy of its associated service account to
+authenticate to the cluster's API server.
+
+For an introduction to service accounts, read [configure service accounts](docs/tasks/configure-pod-container/configure-service-account/).
+
+This guide explains how to add and remove tokens from ServiceAccounts.
+
 {{% /capture %}}
 
-{{% capture body %}}
-## User accounts vs service accounts
-
-Kubernetes distinguishes between the concept of a user account and a service account
-for a number of reasons:
-
-  - User accounts are for humans. Service accounts are for processes, which
-    run in pods.
-  - User accounts are intended to be global. Names must be unique across all
-    namespaces of a cluster, future user resource will not be namespaced.
-    Service accounts are namespaced.
-  - Typically, a cluster's User accounts might be synced from a corporate
-    database, where new user account creation requires special privileges and
-    is tied to complex business processes. Service account creation is intended
-    to be more lightweight, allowing cluster users to create service accounts for
-    specific tasks (i.e. principle of least privilege).
-  - Auditing considerations for humans and service accounts may differ.
-  - A config bundle for a complex system may include definition of various service
-    accounts for components of that system.  Because service accounts can be created
-    ad-hoc and have namespaced names, such config is portable.
-
-## Service account automation
-
-Three separate components cooperate to implement the automation around service accounts:
-
-  - A Service account admission controller
-  - A Token controller
-  - A Service account controller
-
-### Service Account Admission Controller
-
-The modification of pods is implemented via a plugin
-called an [Admission Controller](/docs/reference/access-authn-authz/admission-controllers/). It is part of the apiserver.
```

Comment:
@sftim, what happens to the controller info?

Reply:
Once PR #15374 is ready and merged, I'd like to link to the controller reference information from these service account pages,